voltdbclient/node_conn.go

```diff
@@ -244,6 +244,23 @@ func (nc *nodeConn) loop(writer io.Writer, piCh <-chan *procedureInvocation, res
 			queuedBytes -= req.numBytes
 			delete(requests, handle)
 			if req.isSync() {
+
+				//WARNING: Please do not delete the below commented code it is used to
+				//generate the test samples for deserialization that are found in
+				//test_resources/deserialize
+				//
+				// TODO : Find a better way to generate this?
+				// file := os.Getenv("DES_BATCH")
+				// err = ioutil.WriteFile(file, resp.Bytes(), 0600)
+				// if err != nil {
+				// 	log.Fatal(err)
+				// }
+				// dir := filepath.Dir(file)
+				// err = ioutil.WriteFile(filepath.Join(dir, "handle"),
+				// 	[]byte(fmt.Sprint(handle)), 0600)
+				// if err != nil {
+				// 	log.Fatal(err)
+				// }
```

Comment:
Does it make sense to put this in the test code instead of the driver itself or is the response object already out of context outside of this class?